src/sora-v3.md

```diff
@@ -0,0 +1,128 @@
+- **Supports economic sovereignty**: Balances the role of nation states in managing their domains with the freedom of borderless financial activities.
+
+The SORA v3 Hub Chain will enable anyone, including central banks, to be able to create their own assets on the global SORA v3 platform. For those wanting greater privacy, they can process transactions in a private subnet they control, without sacrificing the security or interoperability of a supranational, decentralized blockchain.
+
+![](/.gitbook/assets/sora-v3-hub-chain-cbdc.png)
+
+Built on [Hyperledger Iroha 2](https://www.lfdecentralizedtrust.org/blog/announcing-hyperledger-iroha-2), the SORA v3 Hub Chain empowers nations, institutions, and individuals by providing a solid infrastructure base to create a robust and inclusive financial system, bringing the best of blockchain technology under one network.
+
+## SORA V3 Tokenomics
+
+The SORA V3 tokenomics were defined over two iterations. The first called for a suspension of TBCD minting while production was funded through the now defunct VXOR token. The second and current iteration uses the KUSD Dollar-pegged synthetic asset to fund production, while implementing buyback and burn of this asset within network transactions. Below is a detailed overview of both iterations.
+
+### First Iteration
+
+![](/.gitbook/assets/sora-v3-2.png)
+
+Economies are like organisms; they are functioning, living entities and must constantly grow and evolve to stay alive. This means that money, the lifeblood of an economy, needs to provide sufficient nourishment to producers while also not increasing beyond the size needed to sustain health.
+
+In SORA v2, the SORA (XOR) token (the main token on the network, used to pay gas fees) had an elastic supply managed by a token bonding curve. The theory behind this was sound — to manage the token supply to meet the economy's needs. However, without sufficient reserves, the SORA v2 economy experienced liquidity shortages.
+
+At the same time, countries worldwide have been debasing their currencies by expanding their monetary bases for non-productive uses. While not directly related to SORA, this is an important narrative for billions of people, and (perhaps intentionally), many people became scared of any expansion in currency supplies, even for productive uses. This meta-narrative has important implications for SORA because we have to work within the confines of the grand narrative created globally, as we are, as of yet, too small to direct it ourselves.
+
+Because of the state of the world and the needs that people all over the world have, SORA v3 must be able to provide adequate funding for producers to target high GDP growth in the SORA economy while also limiting the increase of token supply expansion, given the meta-narrative that has caused people to be fearful of inflation.
```

Comment:
This looks like the beginning of the Second iteration section already. I suggest to move the heading here.

Reply:
added suggestions to add and delete